classes/autoptimizeConfig.php

```diff
@@ -212,8 +217,45 @@ public function show() {
 
 </form>
 </div>
-<style>.autoptimize_banner ul li {font-size:medium;text-align:center;} .unslider-arrow {left:unset;}</style>
-<div class="autoptimize_banner">
+<style>
+.autoptimize_banner {
```

Comment:
Yes of course, I will back to upload the original files unmodified and only apply the changes necessary for the AO-Toolbar.